locales/de-backend.json5

```
@@ -300,6 +341,27 @@
       female: 'Weiblich',
       male: 'Männlich',
       nonbinary: 'Divers',
+
+      Group1: '0 - 4 Jahre',
+      Group2: '5 - 14 Jahre',
+      Group3: '15 - 34 Jahre',
+      Group4: '35 - 59 Jahre',
+      Group5: '60 - 79 Jahre',
+      Group6: 'Über 80 Jahre',
+      Total: 'Gesamt',
     },
   },
+  interventions: {
+    'School closure': 'Schulschließungen',
+    'Face masks & social distancing School': 'Maskenpflicht & Abstandsregeln Schule',
+    'Face masks & social distancing Work': 'Maskenpflicht & Abstandsregeln Arbeit',
+    'Face masks & social distancing Other': 'Maskenpflicht & Abstandsregeln Sonstiges',
+    'Remote work': 'Einführung von Homeoffice-Regelungen',
```

just "Homeoffice-Regelungen" to match "Schulschließungen"?
```suggestion
    'Remote work': 'Homeoffice-Regelungen',
```